lib/matplotlib/backends/backend_gtk3agg.py

```diff
@@ -70,6 +70,7 @@ def draw(self):
         backend_agg.FigureCanvasAgg.draw(self)
         super().draw()
 
+    @_api.delete_parameter("3.5", "args")
     def print_png(self, filename, *args, **kwargs):
```

Comment:
These `args` are passed to the Agg `print_png`; will it double warn, or should you remove it from the latter call?